PySDM/dynamics/condensation/condensation.py

```diff
@@ -31,6 +28,7 @@ def __init__(self, particles_builder: ParticlesBuilder, kappa,
 
         self.do_advection = do_advection
         self.do_condensation = do_condensation
+        self.max_substeps = int(self.particles.dt)
```

Comment:
is it correct? if particles.dt<1, then we will have max_substeps=0

Reply:
yes, because `substeps = max(substeps, max_substeps)`, but I realized that the name of variable is misleading, it should be `min_substeps` (this is number of substeps not length of substeps)